test/integration/tunnel_test.go

```diff
@@ -88,10 +88,16 @@ func testTunnel(t *testing.T) {
 	}
 
 	httpClient := http.DefaultClient
-	httpClient.Timeout = 1 * time.Second
-	resp, err := httpClient.Get(fmt.Sprintf("http://%s", nginxIP))
+	httpClient.Timeout = 5 * time.Second
 
-	if err != nil {
+	var resp *http.Response
+
+	request := func() error {
+		resp, err = httpClient.Get(fmt.Sprintf("http://%s", nginxIP))
+		return err
```

Comment:
AFAIK, For util.Retry to be functional, this error has to be be wraped up into in a util.RetriableError.

Reply:
ah, true, I forgot that small thing :) thanks!